tests/functional/VersionControl/GitVersionControlTest.php

```diff
@@ -148,4 +148,63 @@ public function testGetStagedFilesWithPartiallyStagedFile()
 
         $this->assertSame('test', trim($process->getOutput()));
     }
+
+    public function testGetStagedFilesWithMovedUnrenamedFile()
+    {
+        $testFolderName = 'test_folder';
+
+        $cmd  = 'touch ' . $this->testFileName;
+        $cmd .= ' && echo \'test\' > ' . $this->testFileName;
+        $cmd .= ' && git init';
```

Comment:
oh damn, my (like most of them) docker img with php doesn't have git. test is crashing